public/app/features/plugins/components/AppRootPage.tsx

```diff
@@ -1,16 +1,34 @@
 // Libraries
-import React, { Component } from 'react';
-import { createHtmlPortalNode, InPortal, OutPortal, HtmlPortalNode } from 'react-reverse-portal';
-
-import { AppEvents, AppPlugin, AppPluginMeta, KeyValue, NavModel, PluginType } from '@grafana/data';
+import { AnyAction, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import * as H from 'history';
```

Comment:
don't think this will tree shake properly with the wildcard import
```suggestion
import { Location } from 'history';
```
if we want to avoid clashing with the native `Location`, can always do `import { Location as HistoryLocation} from 'history';`

Reply:
that should work, thanks!